package.json

```diff
@@ -50,11 +50,19 @@
     "cleanup": "mop -v",
     "clean-tests": "pkill -f electron",
     "prepush": "yarn test-peruse",
+    "bump": "standard-version -a",
+    "prerelease": "standard-version --prerelease rc",
     "put-live-net-files-for-osx": "cp ./resources/crust.config node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron\\ Helper.app/Contents/MacOS/Electron\\ Helper.crust.config && cp resources/log.toml /Users/josh/Projects/safe/forks/dev/peruse/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron\\ Helper.app/Contents/log.toml",
     "put-live-net-files-for-linux": "cp ./resources/crust.config node_modules/electron/dist/electron.crust.config && cp resources/log.toml node_modules/electron/dist/log.toml",
     "put-live-net-files-for-windows": "powershell.exe cp ./resources/crust.config node_modules/electron/dist/electron.crust.config; if ($?) { cp resources/log.toml node_modules/electron/dist/log.toml}",
     "rebuild": "npm rebuild --runtime=electron --target=2.0.8 --disturl=https://atom.io/download/atom-shell --build-from-source && cd app && yarn rebuild"
   },
+  "standard-version": {
+    "scripts": {
+      "postbump": "replace $(node -p \"require('./app/package.json').version\")  $(node -p \"require('./package.json').version\") './app/package.json' | [ $(wc -l) == 2 ] ",
```

Comment:
Receiving error when running `yarn bump`:
```
$ standard-version -a
√ bumping version in package.json from 0.7.2 to 0.7.3
√ Running lifecycle script "postbump"
i - execute command: "replace $(node -p "require('./app/package.json').version")  $(node -p "require('./package.json').version") './app/package.json' | [ $(wc -l) == 2 ] "
$(node was unexpected at this time.
[: extra argument '-l)'

Command failed: replace $(node -p "require('./app/package.json').version")  $(node -p "require('./package.json').version") './app/package.json' | [ $(wc -l) == 2 ]
$(node was unexpected at this time.
[: extra argument '-l)'

error Command failed with exit code 1.
```
Bumping version but not generating updated `CHANGELOG`.

Reply:
Hmmm, damn. Windows.

@hunterlester can you try adding a yarn script to test out the command:

`"replace $(node -p "require('./app/package.json').version")  $(node -p "require('./package.json').version") './app/package.json' | [ $(wc -l) == 2 ] "`

And then (as it's this that's failing), try removing `| [ $(wc -l) == 2 ]`

I guess that but is not windows friendly. But let me know either way. Looks like I'll need to write a script to be x-platform secure.

Reply:
@joshuef Bash command substitution, `$( )`, won't work at all in Windows.